<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoIOScreen</class>
 <widget class="QWidget" name="InfoIOScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="coreButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
}

QPushButton:pressed{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Core</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digitalButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
}

QPushButton:pressed{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Digital</string>
   </property>
  </widget>
  <widget class="QPushButton" name="analogButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
}

QPushButton:pressed{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>Analog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rtdButton">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>0</y>
     <width>184</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

	border: none;
	background-color: #00838F;
}

QPushButton:pressed{

	background-color: #006064;
}</string>
   </property>
   <property name="text">
    <string>RTD</string>
   </property>
  </widget>
  <widget class="Line" name="horizontalLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>739</width>
     <height>1</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="verticalLine1">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>0</y>
     <width>1</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="verticalLine2">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>0</y>
     <width>1</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="verticalLine3">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>0</y>
     <width>1</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
